src/map/entities/battleentity.cpp

```diff
@@ -881,6 +884,7 @@ void CBattleEntity::addEquipModifiers(std::vector<CModifier> *modList, uint8 ite
             else
             {
                 m_modStat[modList->at(i).getModID()] += modAmount;
+                m_modStat[modList->at(i).getModID()] += modAmount;
```

Comment:
no

Reply:
that's a typo, I really needed to review this better... sorry, been working on this for like 2 days, brain a bit fried.